presto-main/src/main/java/com/facebook/presto/operator/HttpPageBufferClient.java

```diff
@@ -501,30 +514,6 @@ private static Throwable rewriteException(Throwable t)
         return t;
     }
 
-    private synchronized Duration elapsedErrorDuration()
-    {
-        if (errorStopwatch.isRunning()) {
-            errorStopwatch.stop();
```

Comment:
This code is ancient, and the new Backoff class is much better, so let's just use that